c10/core/TensorImpl.cpp

```diff
@@ -1,16 +1,16 @@
-#include <ATen/core/TensorImpl.h>
+#include <c10/core/TensorImpl.h>
 
 #include <c10/core/Backend.h>
 #include <c10/core/WrapDimMinimal.h>
 #include <c10/util/Optional.h>
 
-namespace at {
+namespace c10 {
 
-Tensor& TensorImpl::grad() {
+at::Tensor& TensorImpl::grad() {
```

Comment:
Err, what. This looks wrong.

Reply:
Yes, but some @ezyang promised me that somebody is already working on removing this method?

Reply:
cc @yf225, maybe this fell of the wagon

Reply:
I am working on moving `grad()` (along with `requires_grad()` and `set_requires_grad()`) into the AutogradMeta struct in TensorImpl, and it would probably happen in the next couple weeks after merging https://github.com/pytorch/pytorch/pull/13827.